Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.8.19 (unreleased)
 
 Language Features:
+* Allow defining custom operators for user-defined value types via ``using {f as +} for Typename;`` syntax.
```

Comment:
```suggestion
* Allow defining custom operators for user-defined value types via ``using {f as +} for Typename global;`` syntax.
```

or just dropping the semicolon as

```suggestion
* Allow defining custom operators for user-defined value types via ``using {f as +} for Typename`` syntax.
```

may be better now, right :-)?

Reply:
Changed to `using {f as +} for T global`.